subprojects/docs/src/docs/userguide/dep-man/01-core-dependency-management/dependency_verification.adoc

```diff
@@ -384,10 +382,12 @@ In practice, it means you need to list the keys that you trust for each artifact
 </component>
 ----
 
-[TIP]
+[WARNING]
 ====
-Gradle supports both full fingerprint ids or long (64-bit) key ids in `pgp`, `trusted-key` and `ignore-key` elements.
-For maximum security, you should use full fingerprints as it's possible to have collisions for long key ids.
+For the `pgp` and `trusted-key` elements, Gradle _requires_ full fingerprint IDs (e.g. `b801e2f8ef035068ec1139cc29579f18fa8fd93b` instead of a long ID `29579f18fa8fd93b`) .
+This minimizes the chance of a https://en.wikipedia.org/wiki/Collision_attack[collision attack].
+
+In `ignore-key` elements, fingerprint ids or long (64-bit) key ids either can be used.
 ====
 
 This effectively means that you trust `com.github.javaparser:javaparser-core:3.6.11` if it's signed with the key `8756c4f765c9ac3cb6b85d62379ce192d401ab61`.
```

Comment:
The example lower of trusting keys globally uses a long ID instead of fingerprint, it needs adaptation as well

Reply:
:thinking:  I see only fingerprints here, or at least I don't understand your issue.

Reply:
This [section](https://docs.gradle.org/current/userguide/dependency_verification.html#sec:trusting-keys-globally), look in the first XML snipper for `javaparser`

Reply:
Should be done.